spec/controllers/admin/users_controller_spec.rb

```diff
@@ -19,4 +19,20 @@
     end
   end
 
+  describe '#lock_account' do
+    it 'it locks the given account' do
+      other_user = FactoryGirl.create :user
+      post :lock_account, id: other_user.id
+      expect(other_user.reload.access_locked?).to be_truthy
+    end
+  end
+
+  describe '#unlock_account' do
+    it 'it unlocks the given account' do
+      other_user = FactoryGirl.create :user
+      post :unlock_account, id: other_user.id
+      expect(not(other_user.reload.access_locked?)).to be_truthy
```

Comment:
When I use: 
```expect(other_user.reload.access_locked?).to be_falsify``` instead of
```expect(!(other_user.reload.access_locked?)).to be_truthy```
I get the error:
http://pastebin.com/aSv3nMv9
If I use the later "expect(!(other_user.reload.access_locked?)).to be_truthy" I don't get any error.